http/server.lua

```diff
@@ -1321,6 +1374,36 @@ local function httpd_start(self)
     return self
 end
 
+-- validate_ssl_opts validates for ssl_opts and return true if at least ssl_cert_file
+-- and ssl_key_file parameteres are not nil.
+local function validate_ssl_opts(ssl_opts)
+    local is_tls_enabled = false
+
+    for ssl_name, ssl_value in pairs(ssl_opts) do
+        if ssl_value == nil then
+            goto continue
+        end
+
+        is_tls_enabled = true
+
+        if type(ssl_value) ~= 'string' then
+            errorf("%s option must be a string", ssl_name)
+        end
+
+        if string.find(ssl_name, 'file') ~= nil and fio.path.exists(ssl_value) ~= true then
+            errorf("file %q not exists", ssl_value)
+        end
+
+        ::continue::
```

Comment:
It just shorted and a little easier to read.
```suggestion
        if ssl_value ~= nil then
            is_tls_enabled = true

            if type(ssl_value) ~= 'string' then
                errorf("%s option must be a string", ssl_name)
            end

            if string.find(ssl_name, 'file') ~= nil and fio.path.exists(ssl_value) ~= true then
                errorf("file %q not exists", ssl_value)
            end
    end
```

Reply:
Refactored.